main/src/test/java/org/mobilitydata/gtfsvalidator/validator/GtfsFrequencyEndRangeValidatorTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.StartAndEndRangeEqualNotice;
+import org.mobilitydata.gtfsvalidator.notice.StartAndEndRangeOutOfOrderNotice;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsFrequency;
+import org.mobilitydata.gtfsvalidator.table.GtfsFrequencyTableLoader;
+import org.mobilitydata.gtfsvalidator.type.GtfsTime;
+
+// FIXME: Cover autogenerated end-range validators by unit tests in "processor".
```

Comment:
@aababilov Could you open an issue on this GitHub repo describing what needs to be done to resolve this FIXME?